src/main/java/com/iexec/worker/result/ResultService.java

```diff
@@ -37,6 +39,7 @@
 import com.iexec.worker.config.BlockchainAdapterConfigurationService;
 import com.iexec.worker.config.WorkerConfigurationService;
 import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.NotImplementedException;
```

Comment:
This import does not seem to be used

Reply:
Thanks! Updated here: https://github.com/iExecBlockchainComputing/iexec-worker/pull/476/commits/5fe56e09aa37d17eed7baad07848d3ee93d26ccb